fuzz/fuzz_helper/src/lib.rs

```
@@ -146,15 +146,39 @@ pub fn process_ctap_any_type(data: &[u8]) -> arbitrary::Result<()> {
     Ok(())
 }
 
+fn setup_env(
+    unstructured: &mut Unstructured,
+    state: &mut CtapState,
+    env: &mut TestEnv,
+) -> arbitrary::Result<()> {
+    if bool::arbitrary(unstructured)? {
+        test_helpers::env::setup_enterprise_attestation(state, env).ok();
```

I'm not sure `.ok()` is the right choice. It can hide actual problems. In which circumstances would this return an error that's not a bug? If we don't know I would start with `.unwrap()` and fix/adapt as we get bug reports from the fuzzer.

---

But this is for fuzzing setup. We arbitrarily setup some customization, then some initial state. I want to decouple the fuzzing of these two, so it's expected that the initial state setup might fail because the customization isn't correct, but it's fine? Otherwise we have to check whether the customization value allows us to do this specific state setup before each setup call and I think that might be too much logic in the fuzzer.

---

I think the problem is that we currently only fuzz for panics and not for correctness, so I see why we don't care about errors. Let's keep it like that for now and hope we rewrite everything when/if we start checking for correctness, because for sure we will miss those silent errors.